content/3. Procedurele informatie/4. Techniek/Workshop Git/2. Git Repositories.md

```diff
@@ -0,0 +1,107 @@
+
+**Stap 2:** Klik linksboven op de knop Current Repository.
+
+**Stap 3:** Klik op Add en selecteer vervolgens Clone Repository.
+
+**Stap 3.1:** Blijf onder het kopje GitHub.com en zoek onder Your Repositories naar de repository die je zojuist hebt aangemaakt.
+
+**Stap 3.2:** Kies onder Local Path waar je de repository wilt opslaan.
+
+**Stap 3.3:** Klik op Clone.
+	De repository is nu als een map op je PC/laptop opgeslagen, bijvoorbeeld in: C:\Users\Gebruikersnaam\Documents\repos.
+
+>[!success] Gefeliciteerd!
+>Je hebt nu je repository gecloned.
+
+
+## Opdracht 3: Eerste push
+>Op dit moment is je repository nog leeg, behalve het .gitignore bestand. Het is tijd om een project of bestand te uploaden.
+
+> [!info] Iets anders proberen?
+> In deze opdracht gebruiken we een .NET Applicatie als voorbeeld. Wil je een ander projecttype gebruiken, of een eenvoudig markdown-bestand? Dat kan ook, ga dan verder naar **Stap 2**.
+
+**Stap 1:** Open Visual Studio (of een andere IDE naar keuze).
+
+**Stap 1.1:** Maak een nieuwe solution aan voor een Console Application.
+
+**Stap 1.2:** Kies de folder die je in de vorige opdracht hebt gecloned als pad voor de solution.
+	Bijvoorbeeld: C:\Users\Gebruikersnaam\Documents\repos
+
+**Stap 2:** Open GitHub Desktop.
+	De bestanden die je zojuist hebt gemaakt, zullen nu zichtbaar zijn in GitHub Desktop onder het kopje ‘Changes’.
+
+> [!warning] Changes
+> Zie je geen changes in GitHub Desktop? Controleer of je de juiste map hebt geselecteerd bij het aanmaken van je project of bestand.
+
+**Stap 2.1:** Vul onderin een ‘summary’ in voor de commit (bijvoorbeeld: Eerste commit).
+	Het is ook mogelijk om een beschrijving toe te voegen. Hierin kun je bijvoorbeeld toelichting geven op de wijzigingen of andere gebruikers taggen.
+
+**Stap 2.2:** Klik op de knop Commit.
+	De wijzigingen zijn nu lokaal opgeslagen als een momentopname van je project. Deze wijzigingen zijn nog niet online zichtbaar.
+
+**Stap 2.3:** Klik op de knop Push origin.
+	Door te pushen, stuur je de commit naar de online repository. Nu zijn de wijzigingen ook zichtbaar en toegankelijk voor anderen die toegang hebben tot je repository.
+
+> [!success] Gelukt!
+> Je hebt je eerste commit gepusht naar Git.
+
+
+## Opdracht 4: Gitignore
+>Om te voorkomen dat tijdelijke of door je IDE gegeneerde bestanden niet continu worden toegevoegd aan je commit gaan we een .gitignore bestand toevoegen.
+
+**Stap 1:** Als je bij het aanmaken van je repository geen .gitignore bestand hebt toegevoegd, kun je deze nu handmatig aanmaken. Maak een nieuw tekstbestand aan met de naam .gitignore in de hoofdmap van je project.
+
+**Stap 1.1:** Voeg de standaard .gitignore regels voor een Visual Studio project toe. Je kunt een voorbeeld .gitignore voor Visual Studio vinden op de [.gitignore repository](https://github.com/github/gitignore/blob/main/VisualStudio.gitignore) of op [Toptal](https://www.toptal.com/developers/gitignore).
```

Comment:
Of via de eerder benoemde cli commando `dotnet new gitignore`, moet je dotnet alleen wel globaal hebben geïnstalleerd, maar dat is vgm standaard. En anders kan het vgm altijd als je git bash gebruikt